### ISSUE TITLE
Show song length in search

### ISSUE THREAD

**Issue by Rawrior** [SEbbaDK/sonica#48]
Some songs may be radio edits but not marked as such, or may be extended cuts. This is completely unknown until the song actually plays. Having the song length along in the search could alleviate this.

**Comment by SEbbaDK** [comment]
I don't think it is available super easily, especially forsomething like deezer :thinking:

**Comment by Rawrior** [comment]
> I don't think it is available super easily, especially for something like deezer :thinking: 

I just checked, and Deezer's API actually exposes a duration field for track objects!  
https://developers.deezer.com/api/track
